packages/server/graphql/public/types/GcalIntegration.ts

```diff
@@ -15,6 +17,45 @@ const GcalIntegration: GcalIntegrationResolvers = {
       .load({service: 'gcal', orgTeamIds: ['aGhostTeam'], teamIds: []})
     if (!globalProvider) return null
     return globalProvider
+  },
+  events: async ({teamId, userId}, {startDate, endDate}, {dataLoader}) => {
+    const gcalAuth = await dataLoader.get('freshGcalAuth').load({teamId, userId})
+    if (!gcalAuth) {
+      return null
+    }
+    const {accessToken: access_token, refreshToken: refresh_token, expiresAt} = gcalAuth
+
+    const CLIENT_ID = process.env.GCAL_CLIENT_ID
+    const CLIENT_SECRET = process.env.GCAL_CLIENT_SECRET
```

Comment:
-1 I don't see this in `env.example` nor in any of the existing env files. let's add that and remember we'll need to add these env variables during next release. also, how do I get these?

Reply:
Ah, these changed since I started working on this! Updated from `GCAL` to `GOOGLE_OAUTH`